dist/enhanced-shutter-card.js

```diff
@@ -1219,12 +1245,12 @@ class EnhancedShutter extends LitElement
     this.pickPoint = event.pageY - parseInt(slide.style.height);
   }
   getShutterPosition(newScreenPosition){
-    let shutterPosition = (newScreenPosition - this.cfg.topOffsetPct()) * (100-this.cfg.offset()) / this.cfg.coverHeightPx();
+    let shutterPosition = (newScreenPosition ) * (100-this.cfg.offset()) / (this.cfg.coverSlatsAreaHeightPx());
     shutterPosition = Math.round(this.cfg.invertPercentage() ?shutterPosition: 100 - shutterPosition);
     return shutterPosition;
   }
   getScreenPosition(pickPoint){
-    let newScreenPosition = Math. round(boundary(pickPoint - this.pickPoint,this.cfg.coverTopPx(),this.cfg.coverBottomPx()));
+    let newScreenPosition = Math. round(boundary(pickPoint - this.pickPoint, 0, this.cfg.coverSlatsAreaHeightPx()));
```

Comment:
Explain this change

Reply:
The behaviour changes due to the margins on the shutter area.